addons/spreadsheet/static/src/list/list_data_source.js

```diff
@@ -72,7 +72,7 @@ export default class ListDataSource extends OdooViewsDataSource {
      * Get the fields to fetch from the server.
      * Automatically add the currency field if the field is a monetary field.
      */
-    _getFieldsToFetch() {
+    _getRelatedFields() {
```

Comment:
I don't think this lies in the PR's scope.
It has anything to do with "fieldsToFetch" of the many2many field.

This is only a basic `search_read` to fetch fields of the model itself.
There's nothing "related" here.

Reply:
My bad :)